docs/podman-info.1.md

```diff
@@ -52,14 +52,14 @@ host:
   kernel: 4.18.7-200.fc28.x86_64
   os: linux
   uptime: 218h 49m 33.66s (Approximately 9.08 days)
-insecure registries:
-  registries: []
 registries:
-  registries:
+  blocked: null
+  insecure: null
+  search:
   - quay.io
   - registry.fedoraproject.org
   - docker.io
-  - registry.access.redhat.com
+  - registry.redhat.io
```

Comment:
Just a FWIW, the latest with v1.8.0 of Buildah actually has:
registries = ['docker.io', 'registry.fedoraproject.org', 'quay.io', 'registry.access.redhat.com', 'registry.centos.org']
I thought we'd changed that to get quay.io up front as shown in this exampel and registries.access.redhat.com to registry.redhat.io?